packages/nitro-protocol/src/index.ts

```diff
@@ -1,29 +1,46 @@
-import NitroAdjudicatorArtifact from '../artifacts/contracts/NitroAdjudicator.sol/NitroAdjudicator.json';
-import TrivialAppArtifact from '../artifacts/contracts/TrivialApp.sol/TrivialApp.json';
-import TokenArtifact from '../artifacts/contracts/Token.sol/Token.json';
-import AssetHolderArtifact from '../artifacts/contracts/AssetHolder.sol/AssetHolder.json';
-import Erc20AssetHolderArtifact from '../artifacts/contracts/ERC20AssetHolder.sol/ERC20AssetHolder.json';
-import EthAssetHolderArtifact from '../artifacts/contracts/ETHAssetHolder.sol/ETHAssetHolder.json';
-import TestNitroAdjudicatorArtifact from '../artifacts/contracts/test/TESTNitroAdjudicator.sol/TESTNitroAdjudicator.json';
-import TestAssetHolderArtifact from '../artifacts/contracts/test/TESTAssetHolder.sol/TESTAssetHolder.json';
+import pick from 'lodash.pick';
+
+import FULLTokenArtifact from '../artifacts/contracts/Token.sol/Token.json';
+import FULLNitroAdjudicatorArtifact from '../artifacts/contracts/NitroAdjudicator.sol/NitroAdjudicator.json';
+import FULLErc20AssetHolderArtifact from '../artifacts/contracts/ERC20AssetHolder.sol/ERC20AssetHolder.json';
+import FULLEthAssetHolderArtifact from '../artifacts/contracts/ETHAssetHolder.sol/ETHAssetHolder.json';
+import FULLTestNitroAdjudicatorArtifact from '../artifacts/contracts/test/TESTNitroAdjudicator.sol/TESTNitroAdjudicator.json';
+import FULLTestAssetHolderArtifact from '../artifacts/contracts/test/TESTAssetHolder.sol/TESTAssetHolder.json';
+
+// https://hardhat.org/guides/compile-contracts.html#artifacts
+const fields = [
+  'contractName',
+  'abi',
+  'bytecode',
+  'deployedByteCode',
+  'linkReferences',
+  'deployedLinkReferences',
+];
+
+interface MinimalArtifact {
+  contractName: typeof FULLNitroAdjudicatorArtifact.contractName;
+  abi: typeof FULLNitroAdjudicatorArtifact.abi;
+  bytecode: typeof FULLNitroAdjudicatorArtifact.bytecode;
+  deployedBytecode: typeof FULLNitroAdjudicatorArtifact.deployedBytecode;
+  linkReferences: typeof FULLNitroAdjudicatorArtifact.linkReferences;
+  deployedLinkReferences: typeof FULLNitroAdjudicatorArtifact.deployedLinkReferences;
+}
 
 export const ContractArtifacts = {
-  NitroAdjudicatorArtifact,
-  TrivialAppArtifact, // TODO do we want to export this?
-  Erc20AssetHolderArtifact,
-  EthAssetHolderArtifact,
-  TokenArtifact, // TODO do we want to export this?
-  AssetHolderArtifact, // TODO do we want to export this?
+  NitroAdjudicatorArtifact: pick(FULLNitroAdjudicatorArtifact, fields) as MinimalArtifact,
+  Erc20AssetHolderArtifact: pick(FULLErc20AssetHolderArtifact, fields) as MinimalArtifact,
+  EthAssetHolderArtifact: pick(FULLEthAssetHolderArtifact, fields) as MinimalArtifact,
```

Comment:
nit: With a helper
```
const minimize = artifact => pick(artifact, fields) as MinimalArtifact
```

this could be a bit neater

```suggestion
  NitroAdjudicatorArtifact: minimize(FULLNitroAdjudicatorArtifact),
  Erc20AssetHolderArtifact: minimize(FULLErc20AssetHolderArtifact),
  EthAssetHolderArtifact: minimize(FULLEthAssetHolderArtifact),
```